lms/security.py

```diff
@@ -66,47 +67,60 @@ def get_route_policy(self, request):
         return self.fallback_policy()
 
 
-class LTISecurityPolicy:
+class _LTIUserBasedSecurityPolicy:
```

Comment:
Base class for all policies that rely on `LTIUser` (which in this PR is just one).